plow.md

```diff
@@ -255,6 +255,13 @@ The use of Narratives and Trade Off Matricies are important aspects of our inter
 
 Working at MayaData will expose you to situations of various levels of difficulty and complexity. This requires focus, and the ability to defer gratification. We value the ability to maintain focus and motivation when work is tough and asking for help when needed.
 
+- Perfect result count -- not a perfect process. Break the rules: fight the law.
+- Assume nothing.<br>
+  Make sure people keep their promises.<br>
+  Push yourselves push others.<br>
+  Stretch the possible.<br>
+- Your job isn't done until <u>the<u> job is done
+
```

Comment:
@visualapps  Thank you for this addition. To keep with the flow of the text, Can you use this suggested modification: 

Put the following at the end of the current paragraph. 
----
We prefer making results count over perfecting the process. We assume nothing and we keep pushing ourselves until the job is done. We keep our promises and make sure our people keep their promises. 
----

Reply:
updated.